ui/src/app/services/conversion/conversion.service.ts

```diff
@@ -446,4 +446,37 @@ export class ConversionService {
     })
     return spName
   }
-}
+  getPGSQLDatatype(spType: string) {
+    switch (spType) {
+      case GoogleSQLDatatype.Bytes:
+        return PGSQLDatatype.Bytea
+      case GoogleSQLDatatype.Float64:
+        return PGSQLDatatype.Float8
+      case GoogleSQLDatatype.Int64:
+        return PGSQLDatatype.Int8
+      case GoogleSQLDatatype.String:
+        return PGSQLDatatype.Varchar
+      case GoogleSQLDatatype.Timestamp:
+        return PGSQLDatatype.TimestampTZ
+      default:
+        return spType
+    }
+  }
+
+  getGoogleSQLDatatype(pgType: string): string {
+    switch (pgType) {
+      case PGSQLDatatype.Bytea:
+        return GoogleSQLDatatype.Bytes
+      case PGSQLDatatype.Float8:
+        return GoogleSQLDatatype.Float64
+      case PGSQLDatatype.Int8:
+        return GoogleSQLDatatype.Int64
+      case PGSQLDatatype.Varchar:
+        return GoogleSQLDatatype.String
+      case PGSQLDatatype.TimestampTZ:
+        return GoogleSQLDatatype.Timestamp
+      default:
+        return pgType
+    }
+  }
```

Comment:
This looks like data that needs to come from an API call to the backend i.e Supply `spType` or List of `spType` and get a `pgSpType` or Map of  <`spType`: `pgSpType`> back.

Currently we are hardcoding the conversion logic in the frontend.

Reply:
Moved this map to be fetched from API